### PR TITLE
When backend error, stop from continuing to query with global ID

### PR DESCRIPTION
**Pull request by nenuadrian**
One issue in this case is that if server errors, but doesn't die, the UI would keep looping, and when executing new analysis it would stack timeouts, all querying for same global ID

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -39,7 +42,7 @@
   </head>
   <body>
     <div class="container-fluid">
-      <h1 class="text-center">3rd Party Auditing Tool</h1>
+      <h1 class="text-center"><img src="icon.png" style="width:100px;"/></h1>
```
Comment (adamread) [inline public/index.html:45]: I think we should leave the title alongside the logo. At the very least the logo needs an alt text.

**Review by adamread — COMMENTED** [review]
I'm against getting rid of the title completely. It seems bad from an A11y standpoint, and the logo doesn't make it as clear what the tool does. 

I'm willing to hear from the rest of the team before we make the final decision though.

File: public/index.html (hunk 1 above)
Comment (nenuadrian) [reply public/index.html]: Done

**Review by adamread — APPROVED** [review]
LGTM

**Pull request merged**